Mathlib/MeasureTheory/Integral/DominatedConvergence.lean

```diff
@@ -0,0 +1,518 @@
+/-
+Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhouhang Zhou, Yury Kudryashov, Patrick Massot
+-/
+import Mathlib.MeasureTheory.Integral.IntervalIntegral
+import Mathlib.Order.Filter.IndicatorFunction
+
+/-!
+## The dominated convergence theorem
+
+This file collects various results related to the Lebesgue dominated convergence theorem
+for the Bochner integral.
+
+## Main results
+- `tendsto_integral_of_dominated_convergence`: the Lebesgue dominated convergence theorem
+- `hasSum_integral_of_dominated_convergence`: the Lebesgue dominated convergence theorem for series
+- `integral_tsum`, `integral_tsum_of_summable_integral_norm`: the integral and `tsum`s commute,
+  if the norms of the functions form a summable series
+- `intervalIntegral.hasSum_integral_of_dominated_convergence`: the Lebesgue dominated convergence
+  theorem for parametric interval integrals
+- `continuous_of_dominated_interval`: continuity of the interval integral w.r.t. a parameter
```

Comment:
```suggestion
- `intervalIntegral.continuous_of_dominated_interval`: continuity of the interval integral w.r.t. a parameter
```
(always give fully qualified names in the file-level summary). Please check the other lemma names too.